package/Dockerfile

```diff
@@ -38,10 +39,6 @@ RUN BUILD_DEPS="make ruby-dev build-essential autoconf automake libtool libsnapp
                      libjemalloc1 \
                      ruby \
     && echo 'gem: --no-document' >> /etc/gemrc \
-    && gem install --file Gemfile \
```

Comment:
Why this removed? Without this command you will get `/run.sh: 23: exec: /usr/local/bin/fluentd: not found` when running container.